career/working-and-collaborating-remotely.md

```diff
@@ -0,0 +1,172 @@
+* **Communicating online** - Obviously, if you are working remotely, you won’t only be communicating by email. It is imperative that you become familiar with the etiquette or “netiquette” when working online and to remain professional during web conferencing.
+  - Every message, email, or call to `@here` on Slack can disturb people’s work, so it’s important to plan your communication
+  - Do people email after hours?  Until you find out, try to send anything outside of normal working hours through scheduled send
+  - In a remote environment, messages that are text only won’t have the social context that one would have if speaking with someone where they can see them, so use care in crafting your email and messages
+  - It pays to be explicit, not implicit, so that the other people you are communicating with know what you intend
+  - For video conferencing, download and try the software ahead of time. You don’t want updates to make you late for your first meeting
+  - Assume the video and audio is on when you log in, reduce distracting noises before hand
+  - Look professional, and wear professional clothing, both for presence, but also in case you have to suddenly run out of the room. Leave the camera on so people get to know you
+
+* **Pairing remotely** - Besides web conferencing tools like Zoom, there are a number of screen sharing tools that work for software development. Again, ask early on if there are any specific tools used to share code, so you can familiarize yourself with them ahead of time. Some software for sharing code include:
+  - [Tuple](https://tuple.app/)
+  - [Screen](https://screen.so/)
+  - [CodeSandbox](https://codesandbox.io/)
+  - [Teletype](https://teletype.atom.io/) for Atom
+  - [Live Share](https://visualstudio.microsoft.com/services/live-share/) for Visual Studio Code
+  - For a more complete list see this list from [designmodo](https://designmodo.com/code-collaboration/)
+* **Asking questions** - In a remote environment, you can’t lean over and ask questions to the person sitting next to you or in the breakroom. To help you get the help you need you should:
+  - Ask your supervisor or team lead what channels you should subscribe to on the company’s chat program
+  - Identify, through looking at various channels, where certain technical and non-technical questions should be directed to. This may be hr@yourcompany.com for some questions, or a slack message for the devops channel for others
+  - Build relationships with your team - see the section below for more info
+
+#### Professionalism and Growth
+* **Being visible for professional growth** - The old adage, “out of sight, out of mind” can easily apply to remote working situations, and it can be hard to make an impression when you are not in the same proximity as your co-workers.  Making an effort to bridge the distance will not only make it easier to work with your team, it will help you make connections you will utilize throughout your career.
```

Comment:
This is a great point. My brother is full-time remote and he said making time to be visible in Slack is an important part of his day.

Reply:
Done, see below